packages/astro/src/core/errors/errors-data.ts

```diff
@@ -900,6 +900,18 @@ See https://docs.astro.build/en/guides/server-side-rendering/ for more informati
 		hint: 'See https://docs.astro.build/en/guides/content-collections/ for more on the `slug` field.',
 	},
 
+	/**
+	 * @docs
+	 * @message The response cannot be altered after it has already been sent.
```

Comment:
No need for this property here, this is only if the actual message is too complex